tests/performance/cfg/cloud-init.yaml

```diff
@@ -0,0 +1,29 @@
+#cloud-config
+user: ubuntu
+password: ubuntu
```

Comment:
we cannot connect to the VM via ssh using a password, because by default PasswordAuthentication=no